morf-testsupport/src/main/java/org/alfasoftware/morf/testing/DatabaseSchemaManager.java

```diff
@@ -330,12 +337,14 @@ public void dropTablesIfPresent(Set<String> tablesToDrop) {
    */
   public void dropAllTables() {
     ProducerCache producerCache = new ProducerCache();
+    if (log.isDebugEnabled()) log.debug("Dropping all tables");
     try {
       Schema databaseSchema = producerCache.get().getSchema();
       ImmutableList<Table> tablesToDrop = ImmutableList.copyOf(databaseSchema.tables());
       List<String> script = Lists.newArrayList();
       for (Table table : tablesToDrop) {
         script.addAll(dialect.get().dropStatements(table));
+        if (log.isDebugEnabled()) log.debug("dropAllTables: " + table.getName());

```

Comment:
Wonder whether this one would read better as "Dropping table: "  + table.getName() seeing as we're logging them out one by one?

Reply:
I see now the pattern you've followed throughout is just starting with the method name